frlzjosh/attendance_checker
# AC-11 | FE: Create state management in the front-end

I created 3 modules that are specific to state in the front-end. 
1. The User module will deal with any state dealing with the person that is currently logged on. Since we have not begun building I did not make many fields. 
2. I created an Instructor State because we have a whole Instructor table and we will need to reference and share his data for the dashboard banner and when knowing which class we will be checking in to
3. I created a class state because of the roster components.

How to test my branch:
1. Go to the login page -> FIll out the username field then click on submit -> You should see the data you entered into the username field appear on the dashboard. It will look like "Welcome *insert name here*"
